packages/office-ui-fabric-react/src/components/Divider/VerticalDivider.Props.ts

```diff
@@ -0,0 +1,16 @@
+import { IVerticalDividerClassNames } from './VerticalDivider.classNames';
+
+export interface IDivider {
+
+}
+
+export interface IVerticalDividerProps {
+
+  /**
+   * Optional callback to access the IDivider interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: (component: IDivider) => void;
+
+  classNames?: Partial<IVerticalDividerClassNames>;
```

Comment:
@dzearing, thoughts on a classNames prop as the solution to the 'I just want to tweak the default styles' problem?